frontend/public/locales/en/translation.json

```diff
@@ -2242,6 +2243,12 @@
     "tab.overview": "Overview",
     "tab.submariner": "Submariner add-ons",
     "tab.userManagement": "User management",
+    "table.acm.channel": "Hub channel",
+    "table.acm.channel.helperText": "The channel that the subscription uses to install Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.consoleUrl": "Hub console URL",
+    "table.acm.consoleUrl.helperText": "The console URL of the Red Hat Advanced Cluster Management for Kubernetes.",
+    "table.acm.distribution": "Hub distribution version",
+    "table.acm.distribution.helperText": "The version of the Red Hat Advanced Cluster Management for Kubernetes.",
```

Comment:
I think we need to remove the word "the" from before RHACM. @swopebe could you also review these, please?

Reply:
I think we can shorten this greatly with a little word change, goodness our product name is long!

**The Red Hat Advanced Cluster Management for Kubernetes console URL** 

**The Red Hat Advanced Cluster Management for Kubernetes version**

(depending on where this is, I don't think we need punctuation, but will let you guys figure that out @KevinFCormier -- it's not actually a complete sentence that warrants a period).

Reply:
Agree to follow our existing standard to keep consistence. Thanks @KevinFCormier and @swopebe 
"the" was removed in my morning time. so the PR is ready for review again.